include/ggml/ggml.h

```diff
@@ -468,6 +468,7 @@ extern "C" {
         GGML_OP_POOL_1D,
         GGML_OP_POOL_2D,
         GGML_OP_UPSCALE, // nearest interpolate
+        GGML_OP_UPSCALE_TO_SHAPE, // nearest interpolate to specified tensor shape
```

Comment:
There is no need to add extra OP, we can reuse the existing `GGML_OP_UPSCALE`

Reply:
fixed